bundle/analyze/superset/superset-init.sh

```diff
@@ -37,52 +34,43 @@ Init Step ${1}/${STEP_CNT} [${2}] -- ${3}
 
 EOF
 }
-ADMIN_PASSWORD="admin"
-# If Cypress run – overwrite the password for admin and export env variables
-if [ "$CYPRESS_CONFIG" == "true" ]; then
-    ADMIN_PASSWORD="general"
-    export SUPERSET_CONFIG=tests.superset_test_config
-    export SUPERSET_TESTENV=true
-    export ENABLE_REACT_CRUD_VIEWS=true
-    export SUPERSET__SQLALCHEMY_DATABASE_URI=postgresql+psycopg2://superset:superset@db:5432/superset
```

Comment:
Are we not setting SQLALCHEMY_DATABASE_URI at all anymore? https://superset.apache.org/docs/installation/configuring-superset says "Make sure to change..."

I think we should add `sqlalchemy_database_uri` as an explicit setting in https://github.com/meltano/hub/pull/530, with a default value of `sqlite:///$MELTANO_PROJECT_ROOT/.meltano/utilities/superset/superset.db`, similar to what we've done for Airflow: https://github.com/meltano/hub/blob/main/_data/meltano/orchestrators/airflow.yml#L15

Note that `export SUPERSET__SQLALCHEMY_DATABASE_URI` suggests that any setting in `superset_config.py` can be set using an env var, prefixed with `SUPERSET__`.

Reply:
@DouweM We don’t set it anywhere explicitly but its defaulted to use that sqlite url in the SUPERSET_HOME directory which is configurable. For this setup the way that users override config settings is by writing them to [superset_config.py](https://superset.apache.org/docs/installation/configuring-superset) so if they had a non-default sqlalchemy uri they would put it in there. Its python so they can access Meltano env vars to template settings as well if they wanted to. I was also imagining they could potentially have multiple configs like `dev/superset_config.py` and `prod/superset_config.py` which get toggled based on the `config_path` Meltano setting using the environments feature.